src/wp-includes/block-supports/align.php

```diff
@@ -33,13 +33,13 @@ function wp_register_alignment_support( $block_type ) {
  * Add CSS classes for block alignment to the incoming attributes array.
  * This will be applied to the block markup in the front-end.
  *
- * @param array         $attributes       Comprehensive list of attributes to be applied.
- * @param array         $block_attributes Block attributes.
  * @param WP_Block_Type $block_type       Block Type.
+ * @param array         $block_attributes Block attributes.
  *
  * @return array Block alignment CSS classes and inline styles.
  */
-function wp_apply_alignment_support( $attributes, $block_attributes, $block_type ) {
+function wp_apply_alignment_support( $block_type, $block_attributes ) {
```

Comment:
Removing `$attributes` isn't backwards compatible. Or have these functions not been released yet? cc. @youknowriad @mcsf

Reply:
No, these were added to trunk two weeks ago. Should be fine to change them, right?

Reply:
Yep no worries. Maybe consider adding `@access private` if they're not intended to be part of the public API.

Reply:
@access private makes sense, yes.